.github/workflows/examples-bouffalolab.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Build example - Bouffalolab BL702
+
+on:
+  push:
+  pull_request:
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.workflow }}-${{ (github.event_name == 'pull_request' && github.event.number) || (github.event_name == 'workflow_dispatch' && github.run_number) || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  bl702:
+    name: BL702
+    timeout-minutes: 90
+
+    runs-on: ubuntu-latest
+    if: github.actor != 'restyled-io[bot]'
+
+    container:
+      image: connectedhomeip/chip-build:latest
+      volumes:
+        - "/tmp/bloat_reports:/tmp/bloat_reports"
+    steps:
+      - uses: Wandalen/wretry.action@v1.0.15
+        name: Checkout
+        with:
+          action: actions/checkout@v3
+          with: |
+            token: ${{ github.token }}
+          attempt_limit: 3
+          attempt_delay: 2000
+      - name: Checkout submodules
+        run: scripts/checkout_submodules.py --shallow --platform bouffalolab
+
+      - name: Set up environment for size reports
+        if: ${{ !env.ACT }}
+        env:
+          GH_CONTEXT: ${{ toJson(github) }}
+        run: scripts/tools/memory/gh_sizes_environment.py "${GH_CONTEXT}"
+
+      - name: Build example BL702 Lighting App
+        timeout-minutes: 30
+        run: |
+          ./scripts/examples/gn_bouffalolab_example.sh lighting-app out/bouffalolab-BL706-IoT-DVK-BL706C-22-light BL706-IoT-DVK
```

Comment:
Please use build_examples.

Why do we have now both examples-buffalolab and examples-bl602?

Reply:
Will change to use build_examples.py for this.

We plan to unified script/configurations for both BL602 and BL702. In next pull request,  examples-bl602.yaml will be deleted, and bl602 jobs will be added in examples-bouffalolab.yaml.

Reply:
Changed to use build_examples.py.